### PR TITLE
feature: support net priority flag

### PR DESCRIPTION
**Pull request by Ace-Tang**
only implment net-priority in client end.

Signed-off-by: Ace-Tang <aceapril@126.com>

### Ⅰ. Describe what this PR did

Only implement net-priority in client end, consider net resource control is not good enough, we not implement it in server client.

### CODE REVIEW

File: apis/swagger.yml [hunk 1]
```diff
@@ -1964,6 +1964,10 @@ definitions:
       QuotaID:
         type: "string"
         description: "set disk quota by specified quota id, if id < 0, it means pouchd alloc a unique quota id"
+      NetPriority:
+        description: "net priority."
```
Comment (allencloud) [inline apis/swagger.yml]: Could you add more description of this field? More detailed content makes more people know more about the great things you are doing.

File: apis/swagger.yml [hunk 2]
```diff
@@ -1964,6 +1964,10 @@ definitions:
       QuotaID:
         type: "string"
         description: "set disk quota by specified quota id, if id < 0, it means pouchd alloc a unique quota id"
+      NetPriority:
```
Comment (shaloulcy) [inline apis/swagger.yml]: NetPriority seem a property of network endpoint. A container may have many endpoints, we may apply different Priorities to different endpoints. Can we consider the situation?

Comment (Ace-Tang) [reply apis/swagger.yml]: Since we not implement in sever, so we just add a flag called `NetPriority` to prepare

File: apis/swagger.yml (hunk 1 above)
Comment (Ace-Tang) [reply apis/swagger.yml]: No, it will be `TODO` things.

File: cli/container.go [hunk 3]
```diff
@@ -72,6 +72,7 @@ type container struct {
 	ulimit         Ulimit
 	pidsLimit      int64
 	shmSize        string
+	netPriority    int64
```
[the remark this answers is not in the document]
Comment (Ace-Tang) [reply cli/container.go]: swagger generate with 
```
      type: "integer"
      default: 0
```

[the remark this answers is not in the document]
Comment (allencloud) [reply cli/container.go]: If you wanna set a specific type, maybe you can consider the following way:
```
      BlkioWeight:
        description: "Block IO weight (relative weight), need CFQ IO Scheduler enable."
        type: "integer"
        format: "uint16"
        x-nullable: false
        minimum: 0
        maximum: 1000
```

**Comment by rudyfly** [comment]
LGTM

**Pull request merged**